framework/include/dirackernels/ReporterPointSource.h

```diff
@@ -46,6 +57,6 @@ class ReporterPointSource : public DiracKernel, public ReporterInterface
   /// weights to scale value by
   const std::vector<Real> & _weight;
 
-  /// map to associate points with their index into the vpp value
-  std::map<Point, size_t> _point_to_index;
+  /// map from an added point to it's weighted value
+  std::unordered_map<Point, Real> _point_to_weightedValue;
```

Comment:
well this is not new to this PR but a map indexed by Points is a terrible idea. 
I made one for transfers and you need quite a bit of extra machinery to avoid floating point precision bugs

Reply:
I thought points got a hash.  @loganharbour

Reply:
I can change it back to an index to values.

Reply:
index from point to values had the same 'problem'

I think I agree we're not getting bad outputs from this here.